src/test/java/com/scottlogic/sorts/SorterComboTest.java

```diff
@@ -0,0 +1,91 @@
+package com.scottlogic.sorts;
+
+import com.scottlogic.UserPost;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class SorterComboTest {
+
+    UserPost userPost1 = new UserPost("Joe Bloggs",
+            OffsetDateTime.of(2020, 1, 5, 7, 12, 3, 0, ZoneOffset.UTC),
+            "Hello World!", 2);
+
+    UserPost userPost2 = new UserPost("Joe Bloggs",
+            OffsetDateTime.of(2020, 1, 3, 8, 53, 34, 0, ZoneOffset.UTC),
+            "Another example post.", 5);
+
+    UserPost userPost3 = new UserPost("Joane Smith",
+            OffsetDateTime.of(2020, 3, 12, 13, 22, 12, 0, ZoneOffset.UTC),
+            "Another example post.", 3);
+
+    @Test
+    public void sort_validNameAndDate_sortedFirstByNameThenDate() {
+        List<UserPost> userPosts = Arrays.asList(userPost1, userPost2, userPost3);
+        AuthorPostSorter sortByAuthor = new AuthorPostSorter();
+        DatePostSorter sortByDate = new DatePostSorter();
+        SorterCombo sortByBoth = new SorterCombo(sortByAuthor, sortByDate);
+        List<UserPost> actualResult = sortByBoth.sort(userPosts, SortOrder.ASC);
+        List<UserPost> expectedResult = Arrays.asList(userPost3, userPost2, userPost1);
+        Assert.assertEquals(expectedResult, actualResult);
+    }
+    @Test
+    public void sort_validInput_sortedFirstByContentThenLikes() {
```

Comment:
I think this name could be a bit better. To me it implies that any valid input should be sorted by content then likes, regardless of how the composite sorter is set up.